src/requirements/Farcaster/FarcasterRequirement.tsx

```diff
@@ -112,7 +118,9 @@ const FarcasterIncludeText = (props: RequirementProps) => {
   return (
     <Requirement image={REQUIREMENTS.FARCASTER.icon.toString()} {...props}>
       <Text as="span">{"Have "}</Text>
-      <DataBlockWithCopy text={data.id} />
+      <Skeleton isLoaded={!!data}>
```

Comment:
`data` will always be loaded (it's the part of the requirement), so we don't need a skeleton here